pgcommitfest/commitfest/views.py

```diff
@@ -1270,6 +1275,28 @@ def cfbot_ingest(message):
                 what="Patch does not need rebase anymore",
             ).save_and_notify(authors_only=True)
 
+    # Similarly, we change the failing_since field using a separate UPDATE
+    failing = branch_status["status"] in ("failed", "timeout") or needs_rebase
+    finished = branch_status["status"] == "finished"
+
+    if "task_status" in message and message["task_status"]["status"] in (
+        "ABORTED",
+        "ERRORED",
+        "FAILED",
+    ):
+        failing = True
+
+    if (failing or finished) and bool(branch_in_db.failing_since) is not failing:
+        print(branch_in_db.failing_since, failing)
```

Comment:
I don't know anything about this code - but I wonder if this is intended to be here?

Reply:
oops yeah